src/AppBundle/Controller/SecurityController.php

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
I agree with... everyone! :D 

The lines of code are important - there is a "perceived complexity" when something has extra lines of code. It's superficial, but, in a sense, people "judge a book bike its cover" by subconsciously seeing a lot of lines of code and thinking that something is complex.

On the other hand, controllers have always behaved differently than other services (where you *do* need to inject objects and use their methods directly). That's always been unfortunate, and has increased the later-parts of Symfony's learning curve (the base controller shortcuts make it easy to get started... but then you pay a bit more of a price later when you need to learn about services, DI, etc).

In this case, for me, I'm ok with it. But, it does strike me as odd that we *still* have a mixture of approaches: sometimes (e.g. `$this->render()`), we're relying on the base controller and using its shortcut methods. But in other cases, we say that you should inject the service you need? If we're moving towards controllers as services, shouldn't we *not* extend the base controller anymore? If we didn't, that would be a big problem, as it would increase the boilerplate code for many things. Unless, we added traits with getter injects (which could be used for both controllers and in services). I'm not sure what the best approach is - this just strikes me as walking half-way between 2 worlds.